spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -296,22 +296,27 @@ An event parameter may have multiple qualifiers.
 public void afterDocumentUpdatedByAdmin(@Observes @Updated @ByAdmin Document doc) { ... }
 ----
 
-Then this observer method will only be notified if all the observed event qualifiers are specified when the event is fired:
+Then this observer method will be notified if the set of observer qualifiers is a subset of the fired event's qualifiers:
 
 [source, java]
 ----
 @Inject Event<Document> documentEvent;
 ...
-documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier()).fire(document);
+documentEvent.select(new UpdatedQualifier(), new ByAdminQualifier(), new ClarificationQualifier()).fire(document);
 ----
 
-Other, less specific, observers will also be notified of this event:
+In the above example the event is fired with `@ByAdmin`, `@Updated`, `@Clarification` and `@Any` qualifiers. `@Any` is added automatically as described in <<event_types_and_qualifier_types>>. The observer qualifiers are `@Updated`, `@ByAdmin` and again `@Any`.
```

Comment:
In fact the @Any mention brings more confusion IMO than @Default mention.
From user perspective @Any is useless regarding observer resolution since firing a qualified event will triggers an unqualified observer (if it has an eligible event type). Adding @Any on one or both side doesn't change this behavior.
OTHO @Default has an effect. If I add @Default to an observer it will be resolved only for events having no qualifier (this use case is not given as an example in the spec).

To make short. Even if @Any is added on Event and Observer (BTW the spec doesn't say anything about this last one) it's totally neutral from user POV, so IMO we should avoid example with @Any to limit user confusion.
Adding an example with @Default on an observer would useful because a lot of user (including myself) wonder how to declare an observer resolved only if the fired event has no qualifier.